packages/web/docs/src/app/layout.tsx

```diff
@@ -0,0 +1,107 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { ReactNode } from 'react';
+import localFont from 'next/font/local';
+import { Layout } from 'nextra-theme-docs';
+import { Head } from 'nextra/components';
+import '@theguild/components/style.css';
+import '../components/navigation-menu/navbar-global-styles.css';
+import '../selection-styles.css';
+import { PRODUCTS } from '@theguild/components';
+import { getDefaultMetadata, getPageMap } from '@theguild/components/server';
+import { Footer } from '../components/footer';
+import { NavigationMenu } from '../components/navigation-menu';
+
+export default function RootLayout({ children }: { children: ReactNode }) {
+  return <HiveLayout>{children}</HiveLayout>;
+}
+
+export const metadata = getDefaultMetadata({
+  productName: PRODUCTS.HIVE.name,
+  websiteName: 'Hive',
+  description:
+    'Fully Open-source schema registry, analytics and gateway for GraphQL federation and other GraphQL APIs',
+});
+
+const neueMontreal = localFont({
+  src: [
+    { path: '../fonts/PPNeueMontreal-Regular.woff2', weight: '400' },
+    { path: '../fonts/PPNeueMontreal-Medium.woff2', weight: '500' },
+    { path: '../fonts/PPNeueMontreal-Medium.woff2', weight: '600' },
+  ],
+});
+
+/**
+ * Alternative to `GuildLayout` for Hive and Hive Gateway websites.
+ * TODO: Move this to `@theguild/components`
+ */
+const HiveLayout = async ({ children }: { children: ReactNode }) => {
+  const [meta, ...pageMap] = await getPageMap();
+
+  const productsPage = pageMap.find(p => 'name' in p && p.name === 'products')!;
+  // @ts-expect-error -- this should be fixed in Nextra, without route, the collapsible doesn't work
+  productsPage.route = '#';
+
+  return (
+    <html
+      lang="en"
+      // Required to be set for `nextra-theme-docs` styles
+      dir="ltr"
+      // Suggested by `next-themes` package https://github.com/pacocoursey/next-themes#with-app
+      suppressHydrationWarning
+      className="font-sans"
+    >
+      <Head>
+        <style>{
+          /* css */ `
+          :root {
+            --font-sans: ${neueMontreal.style.fontFamily};
+          }
+          :root.dark {
```

Comment:
```suggestion
          .dark {
```

Reply:
Without `:root` we have lower specificity. I'm not certain removing it won't change everything.